docs/source/cpp/compute.rst

```diff
@@ -230,10 +234,98 @@ Notes:
   Note that the output can have less than *N* elements if the input has
   less than *N* distinct values.
 
+  The mode kernel is not a proper aggregate (it is actually a vector
+  function, see below).
```

Comment:
Is it important to mention this here?

Reply:
I added it since it would come up eventually if you used the scalar aggregation node through bindings, as you wouldn't be able to call it from there, despite being listed as an "aggregate".

Reply:
I removed it since it's a minor point.